release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -805,6 +805,14 @@ If you disable the `DeploymentConfig` cluster capability, the `deployer` service
 
 For more information, see xref:../installing/cluster-capabilities.adoc#deployment-config-capability_cluster-capabilities[DeploymentConfig capability].
 
+[discrete]
+[id="ocp-4-15-must-gather-storage-default"]
+=== Must-gather storage limit default
+
+ A default limit of 30% of the storage capacity of the container has been added for data collected by the `oc adm must-gather` command. If necessary, you can use the `--volume-percentage` flag to adjust the default storage limit.
```

Comment:
Would it make sense to mention that this limit is 30% percentage of the node's storage that container is running on. 

Also would it make sense to mention what will happen when the limit is exceeded or is it described in somewhere else such as in the link below?

Reply:
Are you saying that the storage limit is not 30% of the container's storage, but 30% of the node's storage? 

What happens if the limit is exceeded is explained in the linked section, I don't think we need to add it to the release note.

Reply:
> Are you saying that the storage limit is not 30% of the container's storage, but 30% of the node's storage?

yes, it is node's storage

Reply:
@ardaguclu How does it look now?